### ISSUE TITLE
NTR: Dolichol levels high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8796]
New term request Dolichol levels high in urine (f357):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000125
- label: Elevated urinary dolichol level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: 
- def: The amount of dolichol in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8796
- synonyms: Dolichol levels high in urine(NONE)
